pwiz_tools/Skyline/EditUI/EditLinkedPeptideDlg.resx

```diff
@@ -126,13 +126,13 @@
     <value>15, 19</value>
   </data>
   <data name="label1.Size" type="System.Drawing.Size, System.Drawing">
-    <value>130, 13</value>
+    <value>127, 13</value>
   </data>
   <data name="label1.TabIndex" type="System.Int32, mscorlib">
     <value>0</value>
   </data>
   <data name="label1.Text" xml:space="preserve">
-    <value>Linked peptide sequence:</value>
+    <value>Linked peptide sequence</value>
```

Comment:
Add the colon back and add a mnemonic. Mnemonics show up in localization and are important for consistency. Please always use them.